tests/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-find_package(Boost 1.55.0 REQUIRED)
+# We move the test files into the testing directory
+file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/hypervolume_test_data/" DESTINATION "${CMAKE_BINARY_DIR}/tests/hypervolume_test_data/")
```

Comment:
I'd still insist on having ``CMAKE_CURRENT_BINARY_DIR`` here. Can you just print the paths of ``CMAKE_BINARY_DIR``, ``CMAKE_CURRENT_BINARY_DIR`` and see if you can alter the copy command accordingly?

The reason I insist is that it's easier to embed pagmo in other projects if we use relative paths for files etc., and the rest of the build system tries to do that.